README.md

```diff
@@ -2,239 +2,293 @@
 
 ![MPS SplashScreen](Documentation/Media/splashscreen.png)
 
-A third-person multiplayer game sample for O3DE, where robots battle one another for dominance in a multi-tiered, under-construction, Starbase
+The MultiplayerSample Project is a third-person multiplayer game built on Open 3D Engine (O3DE), where robots battle one another for dominance in a multi-tiered starbase.
 
-> **_NOTE:_** For Linux, see the [specific setup guide](README_LINUX.md).
+<span style="background-color:#4F3C3C">**NOTE:** For Linux setup, see the guide in [README_LINUX.md](README_LINUX.md)</span>
 
-## Gameplay Overview
-This game sample has players competing to hit the highest score to win. Played over a series of rounds, each round brings greater opportunity for rewards but at higher risks.  Players race to collect gems scattered all over the level.
+## Game overview
 
-Additionally, players have energy shields to protect them. Shooting other player damages their shields and getting hit damages your shields. Once your shields are depleted, you will respawn but at the cost of some of your collected gems.
+In this sample, players compete for the highest score to win. Over a series of rounds, players race around the starbase to collect gems and rack up points. Each player is armed with a laser pistol and protected by a shield. Taking damage from laser blasts depletes the player's shield. Once the shield is depleted, the player respawns at the cost of some of their collected gems.
 
-Do you risk it all to win? 
+Do you risk it all to win?
 
 Game features:
-* 3rd-person character setup.
-* Shooting with laser pistols.
-* A configurable number of rounds; defaults to 3.
-* Configurable gem spawning patterns per round to drive exploration.
-* Support for 1 to 15 players (goal is to raise performance to support 25 players per level).
-* Rich sounds and VFX support.
-* Teleporters to aid exploration.
-* Many points of extensibility.
 
-Note: There's no penalty for respawn if you are knocked off the level or use the teleporters.
+* 3rd-person character setup
+* Weapons (laser pistols) with a reticle, projectile, and visual FX.
+* A configurable number of rounds (default: 3 rounds)
+* Configurable gem spawning patterns per round to drive player exploration
+* Support for 1 to 15 players
+* Rich sounds and Visual FX support
+* Teleporters to aid player exploration
```

Comment:
Nit: Can we:
* Environmental dangers like energy cannons and malfunction shield towers.
* Jump pads to boost players high in the air.

Reply:
Added.